index.html

```diff
@@ -11,7 +21,31 @@ <h4>
        </h4>
        <h5>Hello world</h5>
        <h6>Hello  world</h6>
+       <table >
+           <tr>
+               <td>Name of exam
+               </td>
+               <td>score in exam</td>
+
+           </tr>
+           <tr>
+               <td>SSLC</td>
+               <td>91.5%</td>
+           </tr>
+           <tr>
+               <td>HSC</td>
+               <td>83.5%</td>
+               <tr>
+                   <td>
```

Comment:
@Naveen112001  so on line number 35: are they multiple data in a single row?

Reply:
some edit mistake happened while adding another row